CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
+- Add: save result of apply expression attribute into current context for futher usages
```

Comment:
No associated issue?

(Fine with that, just checking in the case we are missing it... :)